### ISSUE TITLE
Add analytics tracker to web app in the index.html file

### ISSUE THREAD

**Issue by otech47** [setlife-network/website#48]
Google Analytics is the most popular

**Comment by otech47** [comment]
@sofiarm21 @rebecarubio I've added you to the Google analytics account since I had remembered that we are already currently tracking pageviews to the setlife.network site now so we may as well continue building onto that dataset.

Here is the tracking code we need to add

```
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-136322439-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-136322439-2');
</script>

```

**Comment by otech47** [comment]
closed in #64
